content/en/v3/admin/guides/jenkins/example.md

```diff
@@ -77,39 +80,48 @@ controller:
                       id: "15.7.0"
                       npmPackagesRefreshHours: 72
 ```
-File:`/tmp/customPlugs.yaml` - Plugins
+File: `/tmp/values.yaml.gotmpl` - Plugins
 ```
+# These components will merge with default versionStream/charts/jenkinsci/jenkins/values.yaml.gotmpl
 controller:
   installPlugins:
-    - kubernetes:1.29.0
-    - workflow-aggregator:2.6
-    - git:4.6.0
-    - configuration-as-code:1.47
-    - nodejs:1.3.11
+    - kubernetes:3546.v6103d89542d6
```

Comment:
Is this version correct? They dont follow semver?

Reply:
The version was determined after logging into Jenkins app and viewing the manage plugins function. It is a later version of kubernetes plugin that is suitable for the version of Jenkins running (2.332.3). I